### PR TITLE
Remove unused exceptions

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: lib/Doctrine/ORM/ORMException.php [hunk 1]
```diff
@@ -41,38 +41,6 @@ public static function namedNativeQueryNotFound($nativeQueryName)
...
-     */
-    public static function entityMissingAssignedIdForField($entity, $field)
-    {
-        return new self('Entity of type ' . get_class($entity) . " is missing an assigned ID for field  '" . $field . "'. " .
-            'The identifier generation strategy for this entity requires the ID field to be populated before ' .
-            'EntityManager#persist() is called. If you want automatically generated identifiers instead ' .
-            'you need to adjust the metadata mapping accordingly.');
-    }
-
-    /**
```
Comment (Majkl578) [inline lib/Doctrine/ORM/ORMException.php]: This introduced Parse error in master. :/ We really need to fix the test suite asap...

Comment (Majkl578) [reply lib/Doctrine/ORM/ORMException.php]: Fixed ind455a023f8905297f7e8dc9825d1faebb875a48e.

Comment (greg0ire) [reply lib/Doctrine/ORM/ORMException.php]: Oh wow so sorry!

Comment (greg0ire) [reply lib/Doctrine/ORM/ORMException.php]: The test suite is red though, so this is on me: https://travis-ci.org/doctrine/doctrine2/builds/332942097?utm_source=github_status&utm_medium=notification

**Pull request merged**